frontend/context/ServicesProvider.tsx

```diff
@@ -196,8 +217,9 @@ export const ServicesProvider = ({ children }: PropsWithChildren) => {
     <ServicesContext.Provider
       value={{
         services,
-        serviceAddresses,
-        servicesByChain,
+        serviceWallets: serviceAddresses,
```

Comment:
think should call them serviceWallets still

Reply:
Yeah was rushed rename will fix